website/registrations/templates/registrations/register_benefactor.html

```diff
@@ -90,6 +92,66 @@ <h1 class="text-center section-title">{% trans "registration" %} {% trans "Benef
+                        {% blocktrans trimmed %}
+                            This will allow you to sign a Direct Debit mandate, allowing Thalia to withdraw the membership
+                            fees from your bank account. Also, you will be able to use this bank account for future payments
+                            to Thalia via Thalia Pay.
+                        {% endblocktrans %}
+                    </div>
+
+                    <fieldset class="form-group col-12 col-lg-6 direct-debit-fields">
+                        {% bootstrap_field form.initials %}
+                        {% bootstrap_field form.iban placeholder='NL99BANK1234567890' %}
+                        {% bootstrap_field form.bic placeholder='BANKNL2A' %}
+                    </fieldset>
+                    <fieldset id="payments-account-form" class="form-group col-12 col-lg-6 h-100 direct-debit-fields">
+                        <p>
+                            {% blocktrans trimmed %}
+                                By signing this mandate form, you authorise
+                                Study Association Thalia, when your membership
+                                registration will be accepted, to send instructions to
+                                your bank to debit your account in accordance with
+                                the instructions from Study Association Thalia.
+                            {% endblocktrans %}
+                        </p>
+
+                        <p>
+                            {% blocktrans trimmed %}
+                                As part of your rights, you are entitled to a refund
+                                from your bank under the terms and conditions of
+                                your agreement with your bank. A refund must be
+                                claimed within 8 weeks starting from the date on
+                                which your account was debited.
+                            {% endblocktrans %}
+                        </p>
+
+                        <p>
+                            {% blocktrans trimmed %}
+                                Your rights are explained in a statement that you can obtain from your bank.
+                            {% endblocktrans %}
+                        </p>
+
+                        <div class="form-group">
+                            <label for="id_signature">Signature</label>
+                            <input type="hidden"
+                                   name="signature"
+                                   class="form-control"
+                                   placeholder="Signature"
+                                   required=""
+                                   id="id_signature"
+                            >
+
+                            {% include 'payments/includes/signature_canvas.html' %}
+                        </div>
+                    </fieldset>
```

Comment:
Maybe this part should be an imported template? Since it is completely copied in the member registration form.

Reply:
I tried but the styling is subtly different